pkg/build/nodeimage/internal/container/docker/archive.go

```diff
@@ -80,20 +86,31 @@ func GetArchiveTags(path string) ([]string, error) {
 			return nil, err
 		}
 		res = append(res, manifest[0].RepoTags...)
+	} else if hdr.Name == "index.json" {
+		var idx ocispec.Index
+		if err := json.Unmarshal(b, &idx); err != nil {
+			return nil, err
+		}
+		for _, mani := range idx.Manifests {
+			if repoTag, ok := mani.Annotations[ioContainerdImageName]; ok {
```

Comment:
I would expect annotations that are not containerd specific, e.g. `org.opencontainers.image.ref.name` ..?

Reply:
`org.opencontainers.image.ref.name` for tags, not for full `repo + ":" + tag` refs